.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,14 +16,79 @@
 
 ## User-facing changes
 
-<!-- Please describe your changes. -->
+<!--
+Please describe your changes.
+Include before/after screenshots or a short video.
+-->
 
 ## Testing Instructions
 
-<!-- How can the changes in this PR be verified? Relevant for QA  and user acceptance testing. -->
+<!--
+How can the changes in this PR be verified?
+Please provide step-by-step instructions how to reproduce the issue, if applicable.
+Write step-by-step test instructions aimed at non-tech-savvy users, even if the PR is not user-facing.
+-->
+
+### QA
+
+<!-- ignore-task-list-start -->
+* [ ] This is a non-user-facing change and 
+<!-- ignore-task-list-end -->
+
+This PR can be tested by following these steps:
+
+1.
+
+### UAT
+
+<!--
+Sometimes the testing instructions for UAT can differ from the ones for QA.
+-->
+
+<!-- ignore-task-list-start -->
+* [ ] UAT should use the same steps as above.
+<!-- ignore-task-list-end -->
+
+<!--
+If the above checkbox has not been checked, write down all steps necessary for user acceptance testing take to test this PR.
+-->
+This PR can be tested by following these steps:
+
+1.
+
+## Reviews
+
+### Does this PR have a security-related impact?
+
+<!-- Examples: new APIs, changes to KSES, etc.  -->
+
+### Does this PR change what data or activity we track or use?
+
+<!-- Examples: changes to telemetry, new third-party APIs -->
+
+### Does this PR have a legal-related impact?
+
+<!-- Examples: new images with unknown sources, new production dependencies with incompatible licenses -->
+
+## Checklist
+
+* [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
+* [ ] I have tested this code to the best of my abilities
+* [ ] I have verified accessibility to the best of my abilities
```

Comment:
It might be beneficial to include a note about internationalization since RTL and translations are also a priority and often the root of additionally necessary reviews. 

```suggestion
* [ ] I have verified accessibility and internationalization to the best of my abilities
```

Reply:
Ah totally! Especially I should have thought of that since I usually point out i18n stuff! 😄 

I think it could even be its own checklist item to give some more context

```
* [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
```